src/pages/ThorChainLP/Pool/Pool.tsx

```diff
@@ -102,15 +102,19 @@ export const Pool = () => {
   const params = useParams<MatchParams>()
   const translate = useTranslate()
 
+  const poolAssetId = useMemo(() => params.poolAssetId, [params.poolAssetId])
+
+  if (!poolAssetId) throw new Error('poolAssetId is required')
```

Comment:
The only reason why this is undefined is because of `useParams()` returning `Partial<T>`, but if we arrived there, we *do* have an assetId, unless you explicitly navigate to a derp route, which previously looked like this and will now be an error boundary, which I guess is similarly borked?

<img width="1728" alt="image" src="https://github.com/user-attachments/assets/39a6ccc9-167a-4dee-9f87-c59c19024e08" />
<img width="1728" alt="image" src="https://github.com/user-attachments/assets/8e311a99-b36e-460c-a4ae-a1ac4963214c" />